README.md

```diff
@@ -146,7 +146,7 @@ Currently GCC or clang is required (MSVC is not supported). MSYS2 with CLANG64 s
 ## FAQ
 
 Q: Why do you need a very performant version of neofetch?
-> I like putting neofetch in my ~/.bashrc to have a system overwiew whenever i use the terminal, but the slow speed annoyed me, so i created this. Also neofetch didn't output everything correctly (e.g Font is displayed as "[Plasma], Noto Sans, 10 [GTK2/3]") and writing my own tool gave me the possibility to fine tune it to run perfectly on at least my configuration.
+> I like putting neofetch in my ~/.bashrc to have a system overwiew whenever I use the terminal, but the slow speed annoyed me, so I created this. Also neofetch didn't output everything correctly (e.g Font is displayed as "[Plasma], Noto Sans, 10 [GTK2/3]") and writing my own tool gave me the possibility to fine tune it to run perfectly on at least my configuration.
```

Comment:
Also a typo over**v**iew

Reply:
I'm sorry😢, I am actually confused about what you mean here, those "i" should be capitalized

Reply:
Both @LinusDierheimer and I are not native English speaker.  If you are confused by our words, feel free to correct us.

Reply:
Ah ok that makes sense, in English whenever you are referring to yourself by using "i" it must be capitalized because it is referring to you (Native English speakers can also make this mistake very easily) so this part of the merge is actually correct

Reply:
I knew it. However the change `Note: On Windows 10-,` should be reverted or changed to other words that reflects `Windows version older than 10`, and then I'm happy to merge this PR.

Reply:
Oh yeah you are right, it's just 5am where I live so I will fix that as soon as possible 😄